docs/help/Coding-Style-Guideline.md

```diff
@@ -160,47 +157,20 @@ Use either // or /* */ style comments. However, // style is much prefered.
 
 # Coding Style Guideline for Javascript
 
-This coding standard is based on [google javascript coding standard](https://google.github.io/styleguide/javascriptguide.xml)
+The Javascript code follows [AirBnB guideline](https://github.com/airbnb/javascript)
```

Comment:
All the rules are listed in attached link. With examples, rationale behind it, in many languages. I will not make it better, and copy/pasting it here will just add maintenace cost. I explained more here:
https://github.com/Samsung/iotjs/issues/631#issuecomment-281317920